packages/manager/modules/adp/src/service/tracking-progress/translations/Messages_en_GB.json

```diff
@@ -0,0 +1,8 @@
+{
+ "adp_tracking_progress_desc": "Your ADP cluster is in deployment process. Once every step is done, you will be able to manage your entire cluster.",
+ "adp_tracking_progress_title": "Global progression",
+ "adp_tracking_progress_completed_tasks": "Tasks",
+ "adp_tracking_progress_task": "Task",
+ "adp_tracking_progress_get_status_error": "Failed to get progress details for the project: {{message}}",
```

Comment:
Failed to get deployment progress details for analytics data platform {{adp-name}}: {{message}}

Reply:
Dont have the adp name detail here, so its just a simple error message that deployment failed